config/manager/manager.yaml

```diff
@@ -4,18 +4,22 @@ metadata:
   name: controller-manager
   namespace: system
   labels:
-    control-plane: glance-controller-manager
+    control-plane: controller-manager
+    app.kubernetes.io/name: glance-operator
+    app.kubernetes.io/component: glance
 spec:
   selector:
     matchLabels:
-      control-plane: glance-controller-manager
+      app.kubernetes.io/name: glance-operator
```

Comment:
Sorry @abays I didn't have enough time to properly review the patch here. 
Even though it's already merged, I'm +1 on the unique label to select the operator and properly route the requests.